proxstar/__init__.py

```diff
@@ -202,7 +197,19 @@ def list_vms(user_view=None):
                     vms.append(pending_vm)
         else:
             vms = 'INACTIVE'
-    return render_template('list_vms.html', user=user, rtp_view=rtp_view, vms=vms)
+    return render_template('list_vms.html', user=user, user_view=user_view, vms=vms)
```

Comment:
Why was this `rtp_view` variable changed?

Reply:
It existed because the same html file was being used for the pool view and VM view. Tbh it brought more pain than was needed and overall the way variables were being used was confusing. I simplified it by using 2 routes and renaming it to better represent what the thing is

Reply:
Cool, and that's why there's that new route file IG. Works 4 me.